operator/cumulativeSum.go

```diff
@@ -0,0 +1,82 @@
+package operator
+
+import (
+	"encoding/json"
+	"errors"
+	_ "github.com/araddon/qlbridge/qlbdriver"
+	"github.com/liminaab/filtrify/types"
+)
+
+type CumulativeSumOperator struct {
+}
+
+type CumulativeSumConfiguration struct {
+	Column        string `json:"column"`
+	NewColumnName string `json:"newColumnName"`
+}
+
+func (t *CumulativeSumOperator) Transform(dataset *types.DataSet, config string, _ map[string]*types.DataSet) (*types.DataSet, error) {
```

Comment:
I think it would be nicer to have typed config instead, it's pretty hard to know what shape config should be

Reply:
Not sure why config as string is needed, it feels like parsing from JSON (or whatever format) should be in a previous  step and not be defined by the library

Reply:
that is because the middle passing parts doesn't know anything about the config structure. we can also inject new operators to the library. so only the operator itself knows what to cast it's own configuration. they are stored on the same db. so import_manager just passes string config to operators. it doesn't know anything about their structure. it kind of simplifies things

Reply:
okok, makes it a bit harder to use programmatically, but i guess that is not a use case right now